cpp/inc/bond/core/reflection.h

```diff
@@ -200,6 +200,37 @@ const typename method_pointer<Service, Input, Result>::type
     MethodTemplate<Service, Input, Result, method_ptr, metadata_ptr>::method = method_ptr;
 
 
+/// @brief Method description in compile-time schema
+template <
+    typename Service,
+    typename Input,
+    typename Result,
+    const Metadata* metadata_ptr>
+    struct MethodTemplateNoFunc
+{
+    /// @brief Type of the service
+    typedef Service service_type;
+
+    /// @brief Type of the request
+    typedef Input input_type;
+
+    /// @brief Type of the response
+    typedef Result result_type;
+
+    /// @brief Static data member describing method metadata
+    static const Metadata& metadata;
+};
+
+
+template <
+    typename Service,
+    typename Input,
+    typename Result,
+    const bond::Metadata* metadata_ptr>
+    const bond::Metadata&
```

Comment:
Minor: indentation not consistent with `MethodTemplate`

Reply:
Fixed.